### ISSUE TITLE
Add support for custom error

### ISSUE THREAD

**Issue by montyly** [crytic/echidna#715]

**Comment by montyly** [comment]
I added the high priority label - let's figure out how to add this

**Comment by ggrieco-tob** [comment]
It seems it is already a feature from hevm:

https://github.com/dapphub/dapptools/issues/669

I will investigate how to integrate it in echidna directly

**Comment by ggrieco-tob** [comment]
This is already working in echidna 2.0, for instance:

```
$ cat error.sol 
// SPDX-License-Identifier: GPL-3.0
pragma solidity ^0.8.4;

error CustomError(uint256 num);

contract C {
    function fail() public {
       revert CustomError({num: 42});
    }
}

contract Test {
    C c = new C();

    function f() public { }

    function test() public {
      try c.fail() {} catch { assert(false); }
    }
}
$ echidna-test error.sol --test-mode assertion --contract Test
Analyzing contract: /home/g/Code/echidna/error.sol:Test
f(): fuzzing (4672/50000)
test(): failed!💥  
  Call sequence:
    test()

Event sequence: Panic(1), merror Revert CustomError (42) from: C
```

**Comment by ggrieco-tob** [comment]
Already fixed in 2.0.0
